README.md

```diff
@@ -136,6 +136,12 @@ Value of the tag to be filtered against.
 
 `resource_types`: optional list of types kept in the list of resources gathered by tag. If none are specified, then all the resources are kept. All defined metrics must exist for each processed resource.
 
+### Timespan override
+To get the values of the metrics, the exporter uses a default timeframe starting 3 minutes before the time
+of the request and 2 minutes. Some metrics that are not yet gathered will carry 0 values that are not relevant
```

Comment:
This part would need some clarification, but ok for the draft!

Reply:
Ok reworking the sentences then.

Reply:
FYI, here is the description used in the cloudwatch exporter: 
`The newest data to request. Used to avoid collecting data that has not fully converged. Defaults to 600s`

I think you should nor mention the "0", because, I have seen that, for other (most) resource types, 0 is not the last value reported, but they copy the last available one instead, which make a lot more sense

Reply:
Modified